Mathlib/SetTheory/Ordinal/FixedPoint.lean

```diff
@@ -64,9 +63,12 @@ theorem foldr_le_nfpFamily [Small.{u} ι] (f : ι → Ordinal.{u} → Ordinal.{u
 theorem le_nfpFamily [Small.{u} ι] (f : ι → Ordinal.{u} → Ordinal.{u}) (a) : a ≤ nfpFamily f a :=
   foldr_le_nfpFamily f a []
 
-theorem lt_nfpFamily [Small.{u} ι] {a b} : a < nfpFamily f b ↔ ∃ l, a < List.foldr f b l :=
+theorem lt_nfpFamily_iff [Small.{u} ι] {a b} : a < nfpFamily f b ↔ ∃ l, a < List.foldr f b l :=
```

Comment:
I don't like this rename: `_iff` should be omitted if no ambiguity arises

Reply:
I do think this is somewhat ambiguous, considering that `le_nfp` is a different statement.

Reply:
Okay fair